docs/docs/reference/other-new-features/experimental-defs.md

```diff
@@ -279,12 +287,12 @@ class B extends A:
 -  @experimental def f: Int = 2
 ```
 
-### Test frameworks
+## Test frameworks
 
-Tests can be defined as experimental. Tests frameworks can execute tests using reflection even if they are in an experimental class, object or method.
+Tests can be defined as experimental. Tests frameworks can execute tests using reflection even if they are in an experimental class, object or method. Examples:
 
 <details>
-<summary>Examples</summary>
+<summary>Example 1</summary>
```

Comment:
There is only one example, what about:
```suggestion
<summary>Show example</summary>
```

Reply:
@liufengyun `other-new-features/experimental-defs.md` is the Markdown file with the most changes.

Here are two notes about my changes to that file : 
1. The text inside the `<details>`-block is handled *differently* if generating HTML code or generating a PDF document. In particular the show/hide mechanism associated with `<details>`-blocks is stripped out when generating a PDF document.
2. Initially the `<details>`-blocks did include *all code examples* for each of 5 elements of the numbered list.
   ```html
   <details>
     <summary>Examples</summary>
     <pre> // code 1 </pre>
     <pre> // code 2 </pre>
     <pre> ... </pre>
   </details>
   ```
   In the PR version I choose a more modular approach where *each* code example is enclosed in a `<details>`-block with a simple introducing text "Examples : " at the end the preceding text paragraph (as encountered in the rest of the Scala 3 Reference). The end result in HTML looks as follows : 

   <blockquote>
   <details><summary>Example 1</summary><pre>// code example 1</pre></details>
   <details><summary>Example 2</summary><pre>// code example 2</pre></details>
   <details><summary>Example 3</summary><pre>// code example 3</pre></details>
   ...
   </blockquote>

   *Final notes* (feedback is welcome !) : 
    - One possible improvement would be to add a short description for each example, e.g. "Example 1 : bla bla bla" instead of just "Example 1".
    - At some point I also considered the option to *split* some code examples in order to have both examples which *compile correctly* **and** examples which *generate errors*.